crates/fuel-gas-price-algorithm/gas-price-analysis/src/simulation.rs

```diff
@@ -75,7 +75,7 @@ impl Simulator {
             gas_price_factor: NonZeroU64::new(gas_price_factor).unwrap(),
             l2_block_height: 0,
             // Choose the ideal fullness percentage for the L2 block
-            l2_block_fullness_threshold_percent: 50,
+            l2_block_fullness_threshold_percent: 50u8.into(),
```

Comment:
why not `50u64`?

Reply:
```rs
impl From<u8> for ClampedPercentage {
    fn from(value: u8) -> Self {
        Self::new(value)
    }
}
```
I only implemented `into` for `u8`. I considered doing more, but YAGNI.